<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateListCommandWidget</class>
 <widget class="QWidget" name="CreateListCommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Command:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="cbCommands"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pbAddCommand">
     <property name="text">
      <string>Add Command</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableView" name="tvCommands">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbMoveUp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Move Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbMoveDown">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Move Down</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
